test/functional/feature_fee_estimation.py

```diff
@@ -145,8 +145,8 @@ def set_test_params(self):
         self.noban_tx_relay = True
         self.extra_args = [
             [],
-            ["-blockmaxweight=68000"],
-            ["-blockmaxweight=32000"],
+            ["-blockreservedweight=3932000"],
+            ["-blockreservedweight=3968000"],
```

Comment:
To preserve the same behaviour these values should be increased by `DEFAULT_BLOCK_RESERVED_WEIGHT` (8000), I think.

Reply:
Yes thats correct, this increases the weight by 8000.